sofar/update_conventions.py

```diff
@@ -41,28 +43,16 @@ def update_conventions(conventions_path=None, assume_yes=False):
         ``None``, which saves the conventions inside the sofar package.
         Conventions saved under a different path can not be used by sofar. This
         parameter was added mostly for testing and debugging.
-    response : bool, optional
```

Comment:
autsch, i think we need to deprecate the assume_yes first. Do we really need to change the name?

Reply:
Thats actually only a bug in the docs. It has been named `assume_yes` all the time (it was introduced in v1.0.0). So its no renaming